CONFIGURATION.md

```diff
@@ -11,6 +11,33 @@ Here are some Stargate-relevant property groups that are necessary for correct s
+| `stargate.jsonapi.document.limits.max-depth`             | `int` | `8`         | The maximum document depth (nesting).                                                                       |
+| `stargate.jsonapi.document.limits.max-name-length`       | `int` | `48`        | The maximum length of property names in a document for an individual segment.                               |
+| `stargate.jsonapi.document.limits.max-object-properties` | `int` | `64`        | The maximum number of properties any single object in a document can contain.                               |
+| `stargate.jsonapi.document.limits.max-string-length`     | `int` | `16000`     | The maximum length of a single string value in a document.                                                  |
+| `stargate.jsonapi.document.limits.max-array-length`      | `int` | `100`       | The maximum length of a single array in a document.                                                         |
+
+## Operations configuration
+*Configuration for the operation execution, defined by [OperationsConfig.java](src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java).*
+
+| Property                                                | Type  | Default | Description                                                                                                                                                                                         |
+|---------------------------------------------------------|-------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `stargate.jsonapi.operations.default-page-size`         | `int` | `20`    | The default Cassandra page size used for read queries.                                                                                                                                              |
+| `stargate.jsonapi.operations.default-sort-page-size`    | `int` | `100`   | The default Cassandra page size used for read queries that are used for sorting purposes.                                                                                                           |
+| `stargate.jsonapi.operations.max-limit`                 | `int` | `1000`  | // TODO from code this seems like default, not maximum???                                                                                                                                           |
+| `stargate.jsonapi.operations.max-sort-read-limit`       | `int` | `10000` | The maximum amount of documents that could be sorted using the in-memory sorting.                                                                                                                   |
```

Comment:
@maheshrajamani should we call this `stargate.jsonapi.operations.max-document-sort-count`, I could also add explanation that command fails if the limit is reached/broken?

Reply:
Done.